.rubocop.yml

```diff
@@ -71,3 +71,6 @@ Performance/StartWith:
 
 Style/Alias:
   Enabled: true
+
+Style/MultilineArrayBraceLayout:
+  Enabled: true
```

Comment:
Rubocop started failing out of the blue, so I had to fix this as well: https://circleci.com/gh/airbrake/airbrake/688